public/stylesheets/site/2.0/25-media-midsize.css

```diff
@@ -83,5 +95,5 @@ form.filters dl {
 /* 21 userstuff */
 
 #workskin {
-  margin: auto 5%;
-}
\ No newline at end of file
+  margin: auto 1.5%;
```

Comment:
Hmm, I thought I left a comment here, but I guess I didn't! No, this will actually keep the margins looking almost exactly the same as they currently do, since it's 1.5% plus the outer 3.5% instead of 5% plus the outer 0.5em.